Keen.NetStandard/Keen.NetStandard.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
```

Comment:
Was there a specific reason for choosing 10.0.3 or was it just because it's latest stable? I only mention it because we're using 9.0.x in the other projects, I believe. I'm not sure when exactly they started adding support for netstandard, but was just curious if you'd looked into it.